distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -169,33 +167,6 @@ log4j2.appender.securityMain.policies.size.size = 20MB
 log4j2.appender.securityMain.strategy.type = DefaultRolloverStrategy
 log4j2.appender.securityMain.strategy.max = 10
 
-# TODO DDF-3073
```

Comment:
Having this configuration here seems not to work in this file, but works in the log4j2.config.xml.